packages/terra-collapse/tests/nightwatch/collapse-spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
```

Comment:
Do we want to be testing the differences between each nightwatch? Right now we are just generating screenshots except for `Displays content when button is clicked`.